modules/doc/content/help/faq/apptainer.md

```diff
@@ -43,63 +42,66 @@ lengthy prompt)
 
 ## Troubleshoot
 
-!style! halign=left
-You are now operating from within the same container that which your PR is failing. By default,
-your home directory should be available. And by extension, your `~/projects` directory containing
-your project.
-!style-end!
+!style halign=left
+You are now operating from within the same container within which your PR is failing. By default,
+your home directory should be available. And by extension, your `~/cluster_name/projects` directory
+containing your project.
 
-Next, simply put; Clone your application (if you have not). Build it. Run it. Test it. Do whatever
-it is that Civet failed to do.
+!alert! tip
+See [Apptainer Features](help/faq/apptainer.md#features) below if you are operating from your
+`/scratch` directory.
+!alert-end!
+
+Next, simply put: Clone your application (if you have not); build it; run it; and test it. Do
+whatever it is that [!ac](CIVET) failed to do.
 
 ### Troubleshooting Hints
 
-- Be absolutely certain that your project is clean. It might even be best to do exactly what Civet
-  does: Clone your project accordingly just as the `Fetch and Branch` step. Don't use the one you
-  been developing in.
+- Be absolutely certain that your project is clean. It might even be best to do exactly what
+  [!ac](CIVET) does: Clone your project accordingly just as the `Fetch and Branch` step. Don't use
+  the one you been developing in.
```

Comment:
```suggestion
  [!ac](CIVET) does: Clone a clean copy of your project just as in the `Fetch and Branch` step. Don't use
  the one you have been developing in.
```

Reply:
I am not sure I like this change. A clean copy to a lot of users is "cloning it". But Civet may be excluding, or including submodules. Which is why I prefer "clone how Civet clones". Something along those lines.

Do you have another suggesting other than removing "accordingly"? Maybe "identically"?

Reply:
@cticenhour Just pinging on you this, if you're okay with the new change.